from0k2bp.tex

```diff
@@ -430,19 +440,18 @@
 
 A final note (and this will be used a lot): we can extend this yet
 further to create commitments to 2 or even multiple vectors; we just
-need a set of $N$ NUMS base curve points for each vector we're committing
+need a set of $N$~NUMS base curve points for each vector we're committing
```

Comment:
Not sure what you mean here, $N$ NUMS points is surely correct?

Reply:
This is again a formatting thing to add non-breakable space between `N` and `NUMS`. It's good for readability.